.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Feature Request
+about: Solidity language or infrastructure feature requests.
+---
+
+## Prerequisites
+
+- First, many thanks for taking part in the community. We really appreciate that.
+- We realize there is a lot of data requested here. We ask only that you do your best to provide as much information as possible so we can better help you.
+- Support questions are better asked in one of the following locations:
+	- [Solidity chat](https://gitter.im/ethereum/solidity)
+	- [Stack Overflow](https://ethereum.stackexchange.com/)
+- Ensure the issue isn't already reported.
+- Should be reproducible with the latest ``solc`` version.
```

Comment:
This doesn't make sense for a feature request?

Reply:
yup. copy'n'paste was hitting me hard. :-1: